### ISSUE TITLE
Camera added to scene does not display, rotate, do anything ...

### ISSUE THREAD

**Issue by drcmda** [mrdoob/three.js#15929]
##### Description of the problem

I am making [react-three-fiber](https://github.com/drcmda/react-three-fiber), a react renderer for threejs and i am running into a problem that i don't understand. ideally i would like to describe the entirety of threes object3d graph as well as non-object3d's declaratively, so that they can be managed, starting from scene and camera management. For instance, this is how i picture a multi-cam setup:

```jsx
function Content() {
  const camera = useContext(cameraContext)
  const scene = useRef()
  useRender(({ gl }) => void ((gl.autoClear = true), gl.render(scene.current, camera)), true)
  return (
    <scene ref={scene}>
      <mesh>
        <sphereBufferGeometry name="geometry" args={[1, 64, 64]} />
        <meshBasicMaterial name="material" color="white" />
      </mesh>
    </scene>
  )
}

function HeadsUpDisplay() {
  const camera = useContext(cameraContext)
  const scene = useRef()
  useRender(({ gl }) => void ((gl.autoClear = false), gl.clearDepth(), gl.render(scene.current, camera)))
  return (
    <scene ref={scene}>
      <mesh>
        <sphereBufferGeometry name="geometry" args={[0.5, 64, 64]} />
        <meshBasicMaterial name="material" color="black" />
      </mesh>
    </scene>
  )
}

const cameraContext = React.createContext()
function Main() {
  const { width, height } = useThree().size
  const camera = useRef()
  return (
    <>
      <perspectiveCamera
        ref={camera}
        aspect={width / height}
        radius={(width + height) / 4}
        fov={100}
        onUpdate={self => self.updateProjectionMatrix()}
      />
      {camera.current && (
        <cameraContext.Provider value={camera.current}>
          <orbitControls ref={controls} args={[camera.current]} enableDamping />
          <Content />
          <HeadsUpDisplay />
        </cameraContext.Provider>
      )}
    </>
  )
}
```

### The problem

But as soon i start adding cameras to the scene all bets are off, it behaves in the strangest ways. I can go into devtools and set .parent = null and all starts working again.

I've made the following plain threejs sandbox to demonstrate the problem:

* [codesandbox](https://codesandbox.io/s/o7k584q8ry/)

I have googled the hell out of this but only find conflicting information. [Here](https://github.com/mrdoob/three.js/issues/15267) @WestLangley said the camera must not have a parent or its world position must be zero. [Here](https://github.com/mrdoob/three.js/issues/13753) someone says that attached cameras display backwards. [Here](https://github.com/mrdoob/three.js/issues/1046) it says cameras are "automatically" added to the scene, and should be added.

It's all very confusing and i don't know any longer if it's a bug in three or just everything changes once a camera is part of the scene. 

Is it viable for me to target adding and maintaining a camera at all?

##### Three.js version

- [ ] Dev
- [ ] r102
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by WestLangley** [comment]
You have multiple scenes, and you are adding the camera as a child of a scene that is not being rendered.

@mrdoob This is the relevant line in `WebGLRenderer`:
```js
if ( camera.parent === null ) camera.updateMatrixWorld();
```

**Comment by drcmda** [comment]
Exactly, it's part of a scene but since the components opt into rendering themselves (one them doing useRender(..., true) means it takes over the main render loop - the camera will be put by react into the internal scene, but it isn't rendered by three. it could of course be put into "Content's" scene, but 'HeadsUpDisplay' will find it hard to access it in that case, so it would make more sense to store it in the higher-order parent.
